conf/messages.en

```diff
@@ -160,3 +160,10 @@ socialMediaLoss.heading=Has the group made a loss on its social media activity?
 socialMediaLoss.error.required=Select yes if the group made a loss on its social media activity
 socialMediaLoss.checkYourAnswersLabel=Loss made
 socialMediaLoss.change.hidden=Change loss made
+
+reportOnlineMarketplaceAlternativeCharge.title = Are you applying the alternative charge for low or negative UK profit margins to your online marketplace activity?
+reportOnlineMarketplaceAlternativeCharge.heading = Are you applying the alternative charge for low or negative UK profit margins to your online marketplace activity?
+reportOnlineMarketplaceAlternativeCharge.checkYourAnswersLabel = Loss made
+reportOnlineMarketplaceAlternativeCharge.error.required = Select yes if you are applying the alternative charge for low or negative UK profit margins to your online marketplace activity
```

Comment:
@chandraAndem Please check the error key,
```suggestion
report-online-marketplace-alternative-charge.error.required = Select yes if you are applying the alternative charge for low or negative UK profit margins to your online marketplace activity
```

Reply:
Sorry if I change it to hyphen then it will be out of pattern. Instead changed the key in form.